orchagent/portsorch.cpp

```diff
@@ -3411,13 +3475,14 @@ bool PortsOrch::removeBridgePort(Port &port)
     SWSS_LOG_INFO("Flush FDB entries for port %s", port.m_alias.c_str());
```

Comment:
Should this `flushFDBEntries` operation be called before `if (port.m_fdb_count != 0)`